lms/static/js/spec/views/fields_helpers.js

```diff
@@ -63,6 +63,10 @@ define(['backbone', 'jquery', 'underscore', 'common/js/spec_helpers/ajax_helpers
             expect(view.$('.u-field-title').text().trim()).toContain(expectedTitle);
         };
 
+        var expectDropdownSrTitleToContain = function(view, expectedTitle) {
```

Comment:
Nit: I see that this follows other helpers in this file, but I think it would be more flexible to have a helper function that returns the trimmed text. The name of the function wouldn't have to include ``toContain`` so it would be easier to understand. Usage would be something like:

    expect(getDropdownScreenReaderTitle(view)).toContain(expectedTitle);

Then it would be easy to use alternate tests (``toEqual``, ``not.toContain`` etc).